[2025-03-30T02:06:02.472+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-30T02:06:02.500+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:05:00+00:00 [queued]>
[2025-03-30T02:06:02.513+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:05:00+00:00 [queued]>
[2025-03-30T02:06:02.514+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-30T02:06:02.533+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-30 02:05:00+00:00
[2025-03-30T02:06:02.545+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3002) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-30T02:06:02.547+0000] {standard_task_runner.py:72} INFO - Started process 3004 to run task
[2025-03-30T02:06:02.548+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_pipeline', 'fetch_data', 'scheduled__2025-03-30T02:05:00+00:00', '--job-id', '470', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmppw2c_mj8']
[2025-03-30T02:06:02.550+0000] {standard_task_runner.py:105} INFO - Job 470: Subtask fetch_data
[2025-03-30T02:06:02.618+0000] {task_command.py:467} INFO - Running <TaskInstance: ecommerce_data_pipeline.fetch_data scheduled__2025-03-30T02:05:00+00:00 [running]> on host 778b271a0247
[2025-03-30T02:06:02.739+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='ecommerce_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T02:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-30T02:05:00+00:00'
[2025-03-30T02:06:02.742+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-30T02:06:02.742+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-30T02:06:02.743+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-30 02:06:02.501896+00:00
[2025-03-30T02:06:02.744+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline and current dag run status:running
[2025-03-30T02:06:02.744+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-30T02:06:02.746+0000] {Gestioncommands.py:40} INFO - Fetching data from API...
[2025-03-30T02:06:04.724+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host 'my.api.mockaroo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-03-30T02:06:05.547+0000] {Gestioncommands.py:45} INFO - Data fetched successfully
[2025-03-30T02:06:05.552+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-30T02:06:05.563+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-30T02:06:05.563+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_pipeline, task_id=fetch_data, run_id=scheduled__2025-03-30T02:05:00+00:00, execution_date=20250330T020500, start_date=20250330T020602, end_date=20250330T020605
[2025-03-30T02:06:05.587+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-30T02:06:05.588+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-30T02:06:05.589+0000] {logging_mixin.py:190} INFO - Dag name:ecommerce_data_pipeline queued_at:2025-03-30 02:06:00.207027+00:00
[2025-03-30T02:06:05.589+0000] {logging_mixin.py:190} INFO - Task hostname:778b271a0247 operator:PythonOperator
[2025-03-30T02:06:05.622+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-30T02:06:05.661+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T02:06:05.665+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
